server/channels/app/user.go

```diff
@@ -318,6 +318,17 @@ func (a *App) createUserOrGuest(c request.CTX, user *model.User, guest bool) (*m
 		}(ruser.Id)
 	}
 
+	userLimits, appErr := a.GetUserLimits()
+	if appErr != nil {
+		// we don't want to break the create user flow just because of this.
+		// So, we log the error, not return
+		mlog.Error("Error fetching user limits in createUserOrGuest", mlog.Err(appErr))
+	} else {
+		if userLimits.ActiveUserCount > userLimits.MaxUsersLimit {
+			mlog.Warn("User count has exceeded the permissible limit users", mlog.Int("user_limit", userLimits.MaxUsersLimit))
```

Comment:
Here is a suggestion but let's wait for final confirmation from John when he gets back before we merge this 

```suggestion
			mlog.Warn("User created has exceeded the permissible total active users.", mlog.Int("user_limit", userLimits.MaxUsersLimit))
```

Reply:
Done.